doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -181,11 +181,33 @@ Bluetooth Mesh
 
 The Bluetooth Mesh protocol offers the :ref:`ug_bt_mesh_configuring_lpn` feature for optimizing the power consumption of the Bluetooth Mesh devices.
 
+Gazell
+======
+
+Gazell is an asymmetrical protocol where the Device role is power-optimized.
+The exact current consumption will depend on a number of parameters, which are defined statically in the :ref:`nrfxlib:gzll_constants`.
+The most important parameters from the current consumption point of view are:
+
+* :c:macro:`NRF_GZLL_DEFAULT_TX_POWER` - it can be used to find a trade off between communication range and current consumption.
+* :c:macro:`NRF_GZLL_DEFAULT_TIMESLOT_PERIOD`, and :c:macro:`NRF_GZLL_DEFAULT_TIMESLOTS_PER_CHANNEL`, - they can be used to find a trade off between communication latency and current consumption.
+* :c:macro:`NRF_GZLL_DEFAULT_MAX_TX_ATTEMPTS` - it can be used to find a trade off between reliability and current consumption.
+* :c:macro:`NRF_GZLL_DEFAULT_TIMESLOTS_PER_CHANNEL_WHEN_DEVICE_OUT_OF_SYNC` - it can be used to find a trade off between synchronization time and current consumption.
+
 Matter
 ======
 
 To optimize the power consumption of your Matter application, complete the actions listed on the :ref:`ug_matter_device_low_power_configuration` page.
 
+NFC
+===
+
+The |NCS| provides implemenations of two NFC libraries :ref:nrfxlib:`type_2_tag` and :ref:nrfxlib:`type_4_tag`.
+If you want to implement a read-only NFC tag with a short NDEF payload, use the :ref:nrfxlib:`type_2_tag` library, which implements a more lightweight protocol and therefore consumes less energy than the :ref:nrfxlib:`type_4_tag` library.
+For larger payloads the :ref:nrfxlib:`type_4_tag` library may be more power-efficient, as it allows exchanging longer frames which can shorten the overall data exchange.
+The above recommendations are very generic, because the actual performance depends also on the capabilities of the NFC Poller device which reads the tag, so each specific use case needs separate analysis.
```

Comment:
```suggestion
The above recommendations are very generic, because the actual performance depends also on the capabilities of the NFC Poller device that reads the tag, so each specific use case needs separate analysis.
```

Reply:
fixed